.bingo/variables.env

```diff
@@ -12,7 +12,7 @@ BINGO="${GOBIN}/bingo-v0.8.0"
 
 CONTROLLER_GEN="${GOBIN}/controller-gen-v0.12.0"
```

Comment:
Do we also want to bump this to 0.13.0: https://github.com/kubernetes-sigs/controller-tools/blob/v0.13.0/go.mod

Reply:
I'll do this in a follow-up. Thanks for pointing it out!